js/tabSwitch.js

```diff
@@ -0,0 +1,77 @@
+import { tasks } from "./todoapp.js";
+import { initializeTodos } from "./initializeTodos.js";
+import { reassignTodos } from "./reassignTodos.js";
+import { deleteCompletedButton } from "./setCompletedTodosDeletionListener.js";
+
+const showAll = document.querySelector("#all-button");
+const showActive = document.querySelector("#active-button");
+const showCompleted = document.querySelector("#completed-button");
+const taskAddButton = document.querySelector(".task-adding");
+
+function setActiveTab() {
```

Comment:
this function doesn't set active tab actually

callback of tab.addEventListener does